.github/CODEOWNERS

```diff
@@ -15,28 +15,8 @@
 
 ### ADAPTERS
 
-# Adapter interface ("base" + "sql" adapter defaults, cache)
-/core/dbt/adapters              @dbt-labs/core-adapters
-
-# Global project (default macros + materializations), starter project
-/core/dbt/include               @dbt-labs/core-adapters
-
-# Postgres plugin
-/plugins/                       @dbt-labs/core-adapters
-/plugins/postgres/setup.py      @dbt-labs/core-adapters
-
 # Functional tests for adapter plugins
-/tests/adapter                  @dbt-labs/core-adapters
-
-### TESTS
-
-# Overlapping ownership for vast majority of unit + functional tests
-
-# Perf regression testing framework
-# This excludes the test project files itself since those aren't specific
-# framework changes (excluded by not setting an owner next to it- no owner)
-/performance                    @nathaniel-may
-/performance/projects
+/tests/adapter                  @dbt-labs/adapters
```

Comment:
I don't think this is even us anymore, ideally y'all can eventually delete these

Reply:
Agree! Tracked in https://github.com/dbt-labs/dbt-core/issues/9513